src/Makefile.nds.arm7

```diff
@@ -15,7 +15,7 @@ include $(DEVKITARM)/ds_rules
 # all directories are relative to this makefile
 #---------------------------------------------------------------------------------
 BUILD		:=	nds-build
-SOURCES		:=
+SOURCES		:=  borg
```

Comment:
Probably best to drop my suggestion for changing this line since it didn't help with the dependency generation problem